membership-attribute-service/app/models/Attributes.scala

```diff
@@ -92,9 +93,10 @@ object Attributes {
     (__ \ "userId").write[String] and
       (__ \ "tier").writeNullable[String] and
       (__ \ "recurringContributionPaymentPlan").writeNullable[String] and
+      JsPath.writeNullable[Boolean].contramap[Option[Boolean]](_ => None) and // do not serialize highContributor
       (__ \ "oneOffContributionDate").writeNullable[LocalDate] and
       (__ \ "membershipJoinDate").writeNullable[LocalDate] and
-      (__ \ "supporterPlusExpiryDate").writeNullable[LocalDate] and
+      JsPath.writeNullable[LocalDate].contramap[Option[LocalDate]](_ => None) and // do not serialize supporterPlusExpiryDate
```

Comment:
`supporterPlusExpiryDate` is not used, and we decided it's best to not include it in the response